workbench/icons/Makefile.am

```diff
@@ -0,0 +1,20 @@
+icondir = $(datadir)/icons/hicolor/16x16/apps
```

Comment:
Yeah, or use a single Makefile and the `nobase_` prefix, like:

```make
iconsdir = $(datadir)/icons/hicolor
nobase_dist_icons_DATA = \
	16x16/apps/workbench-project.png \
	32x32/apps/workbench-project.png \
	scalable/apps/workbench-project.svg \
	…
```

And of course have the different size in the corresponding folders.

> not sure what the `dist_` adds here since the stuff in `_DATA` should be distributed anyway

No, [`_DATA` is not distributed by default](https://www.gnu.org/software/automake/manual/automake.html#index-_005fDATA), I guess because it's often generated out of templates and the like.

> How are the icons selected by the application?

They are installed as part of the [icon theme](https://developer.gnome.org/gtk3/stable/GtkIconTheme.html#GtkIconTheme.description) so referencing them by name can automatically load them at the most appropriate size.